<!--  Launch file for nearest frontier -->
<launch>

  <!-- Start the Navigator to move the robot autonomously -->
  <node name="Navigator" pkg="nearest_frontier_planner" type="nfp_navigator">
    <param name="update_frequency" value="2"/>
    <param name="map_frame" value="map" />
    <param name="robot_frame" value="base_link" />
  </node>

  <node name="Explore" pkg="nearest_frontier_planner" type="nfp_explore_client" />


</launch>
